### ISSUE TITLE
totalcputime is actually totalrealtime

### ISSUE THREAD

**Issue by ligerzero-ai** [pyiron/pyiron_atomistics#756]
pr.job_table["totalcputime"] is actually the real time consumed by the job (real time between job start and job end)

The standard accounting unit for CPU-time is core-hours on HPCs, so the actual totalcputime should be calculated like so:

no. cores * walltime (hrs)

**Comment by pmrv** [comment]
Correct, but I'm afraid the fix will not be so easy.  We should consider this in the restructuring of the database however.

**Comment by ligerzero-ai** [comment]
While we're here there are also some other issues - in some HPC queueing systems, it is necessary to specify both the number of cores as well as the RAM you would like to utilise. In this case, it is desirable also to extract the maximum RAM used by the process to inform future calculations so as to minimise wait-time on crowded HPC queues.

As a new-user, in the database object, I would expect to find all this basic information about the job.

(i.e. jobid on the HPC, ncpu used, real time elapsed/used, memory used)

Here I note that all this is easily scraped from a HPC using the Slurm jobscheduler using a one-line shell command if you know the jobid on the HPC:

sacct -j 3955179 --format="NCPU,CPUTime,MaxRSS"
     NCPUS    CPUTime     MaxRSS
---------- ---------- ----------
        80 28-16:54:40
        40 14-08:27:20
        80 28-16:54:40
        80 9-02:13:20   1109894K
        80 9-18:24:00   1117012K
        80 5-15:02:40   1074247K
        80 1-04:56:00   1096731K
        80 2-23:57:20

Then, basic arithmetic operations can be performed in bash to get the total job time of a project or otherwise.

I suggest that pyiron could call this sacct command at the end of each job and scrape the data from this.

My prior implementation with bash scripts would just then scrape the data, looping across all folders and outputting it to resources.csv in each subfolder, which can then be iterated over to create another summary of the whole folder.

**Comment by niklassiemer** [comment]
In principle this is true, however, we interact with the queue via the queue adapter pysqa. And that one supports different queue types (well slurm is best supported).

**Comment by pmrv** [comment]
I guess a working solution would be to add a method to the `pysqa` code, that (given the job id) tries to get the resource usage.  If the configured queue system does not define a way to get the resources, it returns `None` or raises an error.  Then whatever is returned to pyiron we store in HDF5 (`job['resources/...']`?).  Maybe we can put an additional object on the job that reads the this information in the HDF5 and presents it in a nice way.
